modules/persistent-storage-cinder-creating-pv.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-cinder.adoc
+
+[id="persistent-storage-cinder-creating-pv_{context}"]
+= Creating the Persistent Volume
+
+You must define your persistent volume (PV) in an object definition before creating
+it in {product-title}:
+
+.Procedure
+
+. Save your object definition to a file.
```

Comment:
This is currently a procedure with only one step. I'd recommend either adding a step afterward that uses `oc create -f <file>` to create the saved file, or changing this to an unordered list.